intranet/apps/eighth/models.py

```diff
@@ -824,28 +826,30 @@ def get_viewable_members_serializer(self, request):
         Returns: QuerySet
 
         """
-        ids = []
         user = request.user
```

Comment:
Couldn't you factor out the duplicative logic from `get_viewable_members_serializer` and `get_viewable_members`?

Reply:
I could, but it would be messy and probably slower. Besides, this isn't too much duplication.